### ISSUE TITLE
Unable to retrieve child resources - SSL_VERSION_OR_CIPHER_MISMATCH

### ISSUE THREAD

**Issue by jbghhp** [microsoft/AzureStorageExplorer#7457]
Already tried system proxy. 

Unable to retrieve child resources

ProducerError:{
  "name": "Electron Net Error",
  "message": "net::ERR_SSL_VERSION_OR_CIPHER_MISMATCH",
  "stack": "Error: net::ERR_SSL_VERSION_OR_CIPHER_MISMATCH\n    at SimpleURLLoaderWrapper.<anonymous> (node:electron/js2c/browser_init:2:49935)\n    at SimpleURLLoaderWrapper.emit (node:events:513:28)",
  "cause": {}
}

**Comment by MRayermannMSFT** [comment]
Hi @jbghhp can you share your OS type and version? E.g. Windows 11, macOS Sanoma, etc.

**Comment by jbghhp** [comment]
Windows 11 Enterprise, 10.0.22621. Version of ASE 1.31.2(94).

**Comment by MRayermannMSFT** [comment]
1. Are you behind a proxy?
2. Are you trying to connect directly to an Azure storage resource, or are you going through a custom domain, private link, anything special like that?
3. Can you check your Windows update?

**Comment by jbghhp** [comment]
Thank you for the response MRayermannMSFT. We did nothing and this started working after 2 weeks of being unavailable. We did have a ticket open with MS, when we got on a call to reproduce the issue for them live everything worked.  We don't use proxy, we didn't run any updates, we do have private link but the other resources on those networks were accessible. The storage was accessible via ADF self-hosted integration runtimes and the azure portal. Sadly I do not have an explanation.
